aws/resource_aws_emr_cluster_test.go

```diff
@@ -2523,161 +3169,478 @@ resource "aws_kms_key" "foo" {
 }
 POLICY
 }
-`, r, r, r, r, r, r, r, r, r)
+`, r, r, r, r, r, r, r, r, r)
```

Comment:
Let’s switch to index notation for the verbs as opposed to positional.

Reply:
I updated this unrelated test to index notation, but I'd really prefer to treat testing cleanup separate from code changes for this resource so we can simplify/fix all the test configurations together (e.g. use `testAccAWSEmrClusterConfigBaseVpc`) and keep the review here concise.

```
--- PASS: TestAccAWSEMRCluster_security_config (404.88s)
```

Reply:
:100: